[TASK] Unify top-level domain of dummy urls (#1600)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/SiteHandling/Basics.rst

      languageId: '2'
      title: Deutsch
      navigationTitle: ''

>       base: 'https://example.de/'

Author: As this refers to "Deutsch" i would stick here to a specific `example.de` instead of a common `example.net`.

>       base: 'https://example.de/'

Reviewer: True but `example.de` does not exist and is none of the registered IANA example domains. If one registers example.de, we are in for trouble again.

Since the whole point of these changes is to normalize the examples to use official example domains we should stick to these.

>       base: 'https://example.de/'

Author: ok

----

The base is the base domain to run a site on. It either accepts a fully qualified URL or a relative segment "/" to react to any domain name.

> It is possible to set a site base prefix to just "/site1" and "/site2" or "example.com" instead of entering a full URI.

Reviewer: ```suggestion
It is possible to set a site base prefix to just :samp:`/site1`, :samp:`/site2` or even :samp:`example.com` instead of entering a full URI.
```

> This allows to have a Site base e.g. example.com to be detected with http and https protocols, although it is recommended to do a HTTP to

Reviewer: ```suggestion
This allows to have a Site base e.g. :samp:`example.com` to be detected with http and https protocols, although it is recommended to do a HTTP to
```

HTTPS redirect either on the webserver level, via a .htaccess rewrite rule, or by adding a redirect in TYPO3.

.. note::
  Please note that this flexibility will introduce side-effects when having multiple sites with mixed configuration settings as Site base:

  + Site 1: /mysite/
  + Site 2: example.com

>   will be unspecific when detecting a URL like example.com/mysite/ and can lead to side-effects.

Reviewer: ```suggestion
  will be unspecific when detecting a URL like :samp:`example.com/mysite/` and can lead to side-effects.
```
